cmd/schema.go

```diff
@@ -0,0 +1,458 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/odpf/salt/printer"
+	stencilv1beta1 "github.com/odpf/stencil/server/odpf/stencil/v1beta1"
+	"github.com/spf13/cobra"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/status"
+)
+
+func SchemaCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "schema",
+		Aliases: []string{"schema"},
+		Short:   "Manage schema",
+		Long: heredoc.Doc(`
+			Work with schemas.
+		`),
+		Example: heredoc.Doc(`
+			$ stencil schema list
+			$ stencil schema create
+			$ stencil schema get
+			$ stencil schema update
+			$ stencil schema delete
+			$ stencil schema version
+		`),
+		Annotations: map[string]string{
+			"group:core": "true",
+		},
+	}
+
+	cmd.AddCommand(createSchemaCmd())
+	cmd.AddCommand(listSchemaCmd())
+	cmd.AddCommand(getSchemaCmd())
+	cmd.AddCommand(updateSchemaCmd())
+	cmd.AddCommand(deleteSchemaCmd())
+	cmd.AddCommand(versionSchemaCmd())
+
+	return cmd
+}
+
+func listSchemaCmd() *cobra.Command {
+	var host string
+	var req stencilv1beta1.ListSchemasRequest
+
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "list all schemas",
+		Args:  cobra.ExactArgs(1),
+		Example: heredoc.Doc(`
+			$ stencil schema list <schema-id>
```

Comment:
let's replace `schema-id` with `namespace-id`

Reply:
This should be a flag right. If not provided it list schemas across all namespaces.

Reply:
Need API support for that. Currently, it returns no schemas if namespace-id is not given.